include/linker/linker-defs.h

```diff
@@ -252,6 +252,17 @@ extern char _ramfunc_ram_size[];
 extern char _ramfunc_rom_start[];
 #endif /* CONFIG_ARCH_HAS_RAMFUNC_SUPPORT */
 
+/* Memory owned by the kernel. Memory region for thread privilege stack buffers,
+ * currently only applicable on ARM Cortex-M architecture when building with
+ * support for User Mode.
+ *
+ * All thread privilege stack buffers will be placed into this section.
+ */
+#ifdef CONFIG_USERSPACE
+extern char _priv_stacks_ram_start[];
```

Comment:
z_ prefix please

Reply:
Well, I'll do this, too, but linker-defs.h is full of linker symbols with "_" or "__" prefixes. This needs clean up then...